### ISSUE TITLE
The question about reproducing the result of SCAN-loss based on the cifar-10.

### ISSUE THREAD

**Issue by caishaotian-szu** [wvangansbeke/Unsupervised-Classification#71]
Hi, it's a gread honor to read your excellent work.

I've got a problem when I reproduced the result of SCAN-loss based on the cifar-10. I have trained a clustering model according to the original codes, by running the scan.py based on the best pretext model and default scan-cifar10 parameters. The epoch is set to even 500, but I just got the result of acc = 62%, unable to reach the best reslut of acc = 81.6% as shown in your paper.

Could you please help me to answer this question? Thank you!

**Comment by wvangansbeke** [comment]
Hi @caishaotian-szu,

I appreciate the interest in our work.

Hard to say what causes it exactly, but the results for CIFAR-10 should be easy to reproduce. If I have to guess, you're probably not loading the pretrained weights (e.g. simclr) correctly. This seems to be quite common. Did you start from the 
provided pretrained weights in the README? Try to verify the missing keys from the dictionary and check if the loading happens correctly. If you trained Simclr yourself, also check if your model is multi-gpu or single-gpu, since the loading in Pytorch is slightly different. If you follow the instructions in the README, you should get around 81%. Most issues arise when people make changes to the code base. So double check that. You can also perform KMeans on the pretrained weights to make sure that they are correct. This should already give you around 66%. Good luck.

**Comment by caishaotian-szu** [comment]
Thank you very much! @wvangansbeke 
I have reproduced the result.
